Sources/TDS/Utilities/TDS+GeneralRules.swift

```diff
@@ -1,11 +1,14 @@
 /// Typealiases for making the translation of the TDS protocol more clear
 /// https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-tds/d2ed21d6-527b-46ac-8035-94f6f68eb9a8
 
+/// BIT
+public typealias Bit = UInt8
+
 /// BYTE
 public typealias Byte = UInt8
 
 /// BYTELEN
-public typealias ByteLen = Byte
+public typealias ByteLen = TDS.Byte
```

Comment:
For now lets leave this unscoped as it doesnt need specification in this context.